src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+spring:
+  application:
+    name: dps-data-compliance
+
+server:
+  port: 8080
+  servlet:
+    context-path: /
+  use-forward-headers: true
+  tomcat:
+    remote-ip-header: x-forwarded-for
+    protocol_header: x-forwarded-proto
+
+graceful:
+  shutdown:
+    enabled: true
```

Comment:
think you've missed `spring-boot-graceful-shutdown` from build.gradle

Reply:
also don't forget to override for dev so that each test doesn't take an extra 30 seconds to shutdown

Reply:
I'll get rid of this for now, the plugin relies on spring-boot-starter-actuator too which I didn't want to add in yet (this PR is supposed to be app insight focussed)